DOCS-1204 - Note about the = operator (#5930)
SumoLogic/sumologic-documentation: docs/cse/rules/cse-rules-syntax.md

   `null = null`

:::note

>  The `=` and `==` functions don’t match against a regular expression or pattern. Instead, use the [`like`](#like) function.

Reviewer: ```suggestion
The `=` and `==` functions do not match against a regular expression or pattern. Instead, use the [`like`](#like) function.
```
